### ISSUE TITLE
How to remove the blue highlighting?

### ISSUE THREAD

**Issue by ajcastro** [mtxr/vscode-sqltools#844]
How can I remove the blue highlight color? This happened when there is no text selection.

![image](https://user-images.githubusercontent.com/4918318/126250645-e34e9181-c72b-4c29-ab5d-161f1ecf7b4f.png)

It disappear when there is a selection:

![image](https://user-images.githubusercontent.com/4918318/126251020-ac39fc55-6d91-4f5e-ab56-9cbdb49ace1a.png)

**Comment by Choens** [comment]
I actually had this exact same problem, but I found an easy "fix". Go to settings and find `sqltools.format`. You will have to edit the settings.json. You will need to create  a variable `highlightQuery`, set to `false`. The default is `true`. This is what mine looks like:

```
"sqltools.format": {
        "reservedWordCase": "lower",
        "indentSize": 2,
        "highlightQuery": false
    },
```

Of course, only the second to last line, `"highlightQuery": false` is specifically relevant, but it shows you what it should look like in context.

Further down in my settings, I also found `"sqltools.highlightQuery": false,`.

Once all of this was set to false, it stopped giving me that highlighting, which I confess, I did not care for either.
